ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md

```diff
@@ -1,2 +1,26 @@
 !!! warning
     This document describes the data model for AVD 4.x. It may or may not work in previous versions.
+
+## IPv6 Standard Access-Lists
+
+### Variables
+
+| Variable | Type | Required | Default | Value Restrictions | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| [<samp>ipv6_standard_access_lists</samp>](## "ipv6_standard_access_lists") | List, items: Dictionary |  |  |  | IPv6 Standard Access-Lists |
+| [<samp>&nbsp;&nbsp;- name</samp>](## "ipv6_standard_access_lists.[].name") | String | Required, Unique |  |  | ipv6_access_list_name |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;counters_per_entry</samp>](## "ipv6_standard_access_lists.[].counters_per_entry") | Boolean |  |  |  |  |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;sequence_numbers</samp>](## "ipv6_standard_access_lists.[].sequence_numbers") | List, items: Dictionary | Required |  |  |  |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- sequence</samp>](## "ipv6_standard_access_lists.[].sequence_numbers.[].sequence") | Integer | Required, Unique |  |  | sequence_id |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;action</samp>](## "ipv6_standard_access_lists.[].sequence_numbers.[].action") | String | Required |  |  | action as string |
```

Comment:
The links seems borken they redirect to `https://github.com/aristanetworks/ansible-avd/blob/577523278a388f4044215f9ab765f51a13261658/ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input%20Variables.md##` when looking on Github.

Reply:
these are not real linked but used to display the path only so resolving this.